tests/conftest.py

```diff
@@ -528,6 +528,23 @@ def test_perf_signature_2(test_perf_signature):
     )
 
 
+@pytest.fixture
+def test_perf_signature_3(test_perf_signature):
```

Comment:
Could you give this fixture a more meaningful name? The "3" suffix is totally opaque.